digiroad2-geo/src/test/scala/fi/liikennevirasto/digiroad2/GeometryUtilsSpec.scala

```diff
@@ -301,6 +301,35 @@ class GeometryUtilsSpec extends FunSuite with Matchers {
     mValue should be(1.0)
   }
 
+  test("Project stop location to closest point of geometry if point lies next to geometry") {
```

Comment:
mihis näitä testejä tarvitaan? oliko nää sun debug testejä?

Reply:
Joo, nää voisi oikeastaan poistaa turhana, testailin näillä sitä toista m-arvo laskutapaa

Reply:
Poistettu